### PR TITLE
Update QUEST_en.tsv

### PR DESCRIPTION
**Pull request by Sourpusss**
Lines 2000-2034

This needs to be merged with the new file, "Quest.tsv"

### CODE REVIEW

File: QUEST_en.tsv [hunk 1]
```diff
@@ -1997,38 +1997,38 @@ QUEST_20150511_001995	(Everything is scribbled so badly so I can't recognize whe
...
-QUEST_20150511_001999	(there are words that I can barely read){nl}.. Euros could not hold his breath anymore and he drank the spore and died.{nl}Mushwort hunts people with the most brutal method I have ever seen.{nl}It walks around our gathering places and shakes out the mushrooms that would release spores.{nl}
-QUEST_20150511_002000	We never know when it would release the spores, but if we don't stop it, the whole camp would be annihilated.{nl}But, I have no way to block it. I can't help stop coughing..
-QUEST_20150511_002001	The training officer completed the training at the camp in woods of attached bridge in accordance with the order to prepare for future disasters. I hereby grant this certificate. 
+QUEST_20150511_001999	(There are words that I can barely read){nl}.. Euros could not hold his breath anymore so he inahled a spore and died.{nl}Mushwort hunts people with the most brutal method I have ever seen.{nl}It walks around our gathering camps and shakes out the poisonous mushrooms.{nl}
```
Comment (Sunaries) [inline QUEST_en.tsv]: inahled -> inhaled

File: QUEST_en.tsv [hunk 2]
```diff
@@ -1997,38 +1997,38 @@ QUEST_20150511_001995	(Everything is scribbled so badly so I can't recognize whe
...
-QUEST_20150511_002003	For the first step, you just should recharge the magical essence I gave to you. Look for the place where the magic is condensed at Pavydo mixed forest.{nl}But, you won't be able to use the magical essence right away. Since it also absorved dirty energies. So please collect the saliva of Operors that has the purification effect.{nl}At the wasteland of Renistu, the altar of purification should be there. Purify the magical essence with the saliva of Operors there.{nl}
+QUEST_20150511_002003	For the first step, you just should recharge the magical essence I gave to you. Look for the place where the magic is condensed at Pavydo Mixed Forest.{nl}But, you won't be able to use the magical essence right away, since it also absorbed dirty energies. So please collect the purifiying saliva of Operors.{nl}The altar of purification should be in the Wasteland of Renistu. Purify the magical essence with the Operors' saliva,there.{nl}
```
Comment (Sunaries) [inline QUEST_en.tsv]: purifiying -> purifying

File: QUEST_en.tsv [hunk 3]
```diff
@@ -1997,38 +1997,38 @@ QUEST_20150511_001995	(Everything is scribbled so badly so I can't recognize whe
...
+QUEST_20150511_002015	With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. Trees should be hid inside the forest. {nl}Rukas Plateau and Plateau of Rex.. I feel like I could touch something with my fingertips.{nl}I am sure that the goddess has sent this lad to me. May the blessings of the goddess always be with us! 
+QUEST_20150511_002016	Some poles unlock the tomb, and some altars lock the tomb.{nl}The magical energy that surrounds the tomb moves around the key.{nl}But, no one seems to be followed.{nl}We are not chosen.
+QUEST_20150511_002017	Curses disturbe the rules of the universe that are being controlled by the goddess with humanity's greed.{nl}These rules in the human world get influenced by the personalities of each human, their abilities, and their potential power.{nl}The cases when these abilities get released depend on the conditions above.
```
Comment (Sunaries) [inline QUEST_en.tsv]: disturbe -> disturb

File: QUEST_en.tsv [hunk 4]
```diff
@@ -1997,38 +1997,38 @@ QUEST_20150511_001995	(Everything is scribbled so badly so I can't recognize whe
...
+QUEST_20150511_002018	According to the old writing left on the great cathedral, this is like secretly using a small part of the goddesess' abilities.{nl}Some people say that by seducing the weak minds of humans, demons used these minds to form the curse.{nl}{nl}May the goddess bless!
```
Comment (Sunaries) [inline QUEST_en.tsv]: goddesess' -> goddesses'

File: QUEST_en.tsv [hunk 5]
```diff
@@ -1997,38 +1997,38 @@ QUEST_20150511_001995	(Everything is scribbled so badly so I can't recognize whe
...
+QUEST_20150511_002019	There are two ways to release the curse.{nl}One way is to remove the curse with greater power and the  other is to eliminate the curse's caster.{nl}
```
Comment (Sunaries) [inline QUEST_en.tsv]: There is an extra space between 'the' and 'other'

**Comment by imcgames** [comment]
Manually updated due to the changed file names

**Pull request closed without merging**